tox.ini

```diff
@@ -52,9 +52,12 @@ show-source = True
 # E123, E125 skipped as they are invalid PEP-8.
 # N802 skipped (function name should be lowercase)
 # N806 skipped (variable in function should be lowercase)
-ignore = E123,E125,N802,N806
+# F401 skipped (imported but unused) after verifying current usage is valid
+# W503 skipped line break before binary operator
+# C901 skipped: 'MLaunchTool.init' is too complex
+ignore = E123,E125,N802,N806,F401,W503,C901
 builtins = _
-exclude=.venv,.git,.tox,dist,*lib/python*,*egg,*figures/*,__init__.py
+exclude=.venv,.git,.tox,dist,*lib/python*,*egg,*figures/*,__init__.py,build/*,setup.py,mtools/util/*,mtools/test/test_*
```

Comment:
Do we want to ignore testing for `mtools/util/*`? Perhaps exclude for now, but raise an issue to fix violations at some point in the future?

Reply:
Ignoring for now, but will re-add in future via #655.